### ISSUE TITLE
Entropinnyum weird behaviour when using multiple

### ISSUE THREAD

**Issue by Joly0** [Vazkii/Botania#4751]
### Mod Loader

Forge

### Minecraft Version

1.20.1

### Botania version

1.20.1-446-FORGE

### Modloader version

47.3.7

### Modpack info

Ozone Skyblock Reborn

### The latest.log file

Not important

### Issue description

I have rebuild an Entropinnyum design i have made in another Modpack (OPSkies 2, 1.19.2) where the design works absolutely perfect. But in the newer Version it doesnt anymore.
First the screenshot of the setup from OPSkies2:
![grafik](https://github.com/user-attachments/assets/a119637e-170c-4764-89f1-7ea145a12c19)
And a screenshot of the setup in Ozone Skyblock Reborn:
![grafik](https://github.com/user-attachments/assets/154626d0-092b-4be0-a538-3988f2dabdb3)

In both setups there is a hole right behind the Entropinnyum that leads to an obsidian tunnel which has its last block 12 blocks away from the flower, which should be perfectly in range for it to recognize the explosion, eat it up and generate mana.
For both setups i have setup timers set to 120 ticks so the flowers and mana spreaders are empty before the next tnt gets deployed.
I have also adjusted the setup on OPSkies2 so each mana pool level is read and if there is enough free space for another load, only the correct dispenser drops another tnt, no other dispenser gets activated and only that particular Entropinnyum eats it up, no other.

In OPSkies2 all Entropinnyums correctly get filled up by their own dispenser/tnt combo, but in Ozone Skyblock Reborn it is very randomly which Entropinnyum gets the tnt. For example in this screenshot i only drop tnt from the far right dispenser, but the Entropinnyum that should get triggered doesnt get triggered:
![grafik](https://github.com/user-attachments/assets/24f79600-a8b3-4686-9f00-61e291becbf0)
Here as a comparision in OPSkies2 i have removed and placed the mana pool again so it triggers the dispenser, and only that one Entropinnyum gets the explosion trigger and takes up the mana:
![grafik](https://github.com/user-attachments/assets/15f24a39-1dff-4129-8aee-92f5ed0af119)

This is extremely weird behaviour and i have no idea why that is.

Other than the wireless transmitter and receiver aswell as the comparator (which i havent build in Ozone Skyblock Reborn because of this issue) both setups are exactly identical. The distance from the flower to the bottom of the obsidian tunnel is same, the dispenser and tnt setup is same, the timers are the same.

### Steps to reproduce

I have no idea what steps this would include, i guess download Ozone Skyblock Reborn, build a similar setup and try it out, compare it to the same setup in OPSkies2

**Comment by Joly0** [comment]
Or another screenshot to show this in OPSkies2:
![grafik](https://github.com/user-attachments/assets/3c9aa0c5-8eb8-41d7-9425-cc0d935ed102)
I still have the far left mana pool replaced with an empty one aswell as the fourth one and only those two trigger tnt behind them and only those two fill up correctly

Tldr, i think something might have changed between 1.19 and 1.20 so the entropinnyum is less "nearest first" and more "randomly"

**Comment by Joly0** [comment]
Another screenshot from Ozone Skyblock Reborn:
![grafik](https://github.com/user-attachments/assets/599f4fca-a191-4e95-9c56-7d079f83b483)

10 Primed tnt only trigger 8 Entropinnyum, the other 2 tnt just explode while the 2 Entropinnyums on the right are doing nothing, the flowers itself aswell as the mana spreader are empty.

**Comment by TheRealWormbo** [comment]
Does this also happen with just Botania alone? And if not, can you provide a smaller set of mods such that it does?

**Comment by TheRealWormbo** [comment]
As another note, Entropinnyum range is reasonably large a cube, as such there is no guarantee any particular flower within that range will handle the explosion. There's no "closest flower wins" mechanic, so I'm not sure there's an actual bug here.

**Comment by TheRealWormbo** [comment]
Going to assume this works as intended due to lack of further information. Please respond if you have additional details.
